### DISCUSSION TITLE
Em relação ao projeto fiquem a vontade para dar dicas, melhorias e atualizações de codigo. toda informação é bem vinda. alguma duvida?

### DISCUSSION THREAD

**Discussion by Adriano-Nelex** [Adriano-Nelex/landing-page#2] [Q&A] [answered]
Em relação a frameworks CSS, quais vocês gostam de utilizar, Bootstrap ou Tailwind?

**Comment by AdrianoWEBFernandes** [comment, marked as answer]
Bootstrap For me is the best framework, because it's more flexibility in modes responsives and the codes are very simple to understand.

**Reply by Adriano-Nelex** [reply]
Thank you, i agree with you, my jobs i usually in my projects. it's a great framework.
